### DISCUSSION TITLE
[Algorithm] 백준 11727 2×n타일링2 (C++)

### DISCUSSION THREAD

**Discussion by llddang** [BDD-CLUB/challenge-every-week-posting#31] [3. CS(컴퓨터 공학) 관련 글 모음]
헤헷 적을게 없어서 알고리즘 문제 풀이만 주구장창 올리게 될 것 같은데...
빨리 한국돌아가서 나도 예쁘고 재밌는거 만들고 싶다!!!!!!!!!!!!!!!!!!!!!!!!!

[블로그~!](https://llddang-blog.tistory.com/6)

**Comment by pnudpwls1103** [comment]
이쁘고 머싯고 재밌는거!!! 뿌슝빠슝
알고리즘 공부하는 따은이 멋져

**Comment by shkisme** [comment]
알고리즘 공부까지 하는 당신... 그저 갓생 ㅠㅠ
(나는 알고리즘 언제하지?)
